### PR TITLE
2023-01-12-Lendhub.md

### PR DESCRIPTION
**Pull request by svg-arch**

### CODE REVIEW

File: content/attacks/posts/2023-01-12-Lendhub.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- Lendhub Hacker wallet: [0x9d0163e76bbcf776001e639d65f573949a53ab03](https://etherscan.io/address/0x9d0163e76bbcf776001e639d65f573949a53ab03)
+
+## Losses
+
+Lendhub lost approximately $6 million.
+
+## Timeline
+
+- **January 12, 2023:**  Lendhub is hacked
```
Comment (JediFaust) [inline content/attacks/posts/2023-01-12-Lendhub.md:25]: Provide malicious transaction with exact time if possible, the sentence looks incomplete.

File: content/attacks/posts/2023-01-12-Lendhub.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+- **January 13, 2023:**  [Lendhub announced the hack on their platform and the extent of the losses, with around $6 million stolen](https://twitter.com/LendHubDefi/status/1613846541651030018)
+- **January 13, 2023:**  [The attacker has made 11 transactions totaling 1,100 ETH to TornadoCash](https://twitter.com/SlowMist_Team/status/1613906600279900162)
+- **February 26, 2023:**  [The attacker has deposited another 2,415.4 ETH into Tornado.cash](https://twitter.com/PeckShieldAlert/status/1630027178426642432)
+
+## Security Failure Causes
+
+The main reason for the Lendhub attack was the failure to remove a deprecated token from the market properly. This vulnerability allowed attackers to exploit the protocol's smart contracts and steal funds.
```
Comment (JediFaust) [inline content/attacks/posts/2023-01-12-Lendhub.md:32]: Make bullet point as in other articles, provide little more details and shorten or remove last sentence

**Review by JediFaust — CHANGES_REQUESTED** [review]

File: content/attacks/posts/2023-01-12-Lendhub.md [hunk 3]
```diff
@@ -22,11 +24,13 @@ Lendhub lost approximately $6 million.
...
+- **January 12, 2023:** The exploiter then interacted with the attack contracts multiple times, which drained assets from the new Lendhub HECO market.
+- **January 13, 2023:** [Lendhub announced the hack on their platform and the extent of the losses, with around $6 million stolen](https://twitter.com/LendHubDefi/status/1613846541651030018)
+- **January 13, 2023:** [The attacker has made 11 transactions totaling 1,100 ETH to Tornado.cash](https://twitter.com/SlowMist_Team/status/1613906600279900162)
+- **February 26, 2023:** [The attacker has deposited another 2,415.4 ETH into Tornado.cash](https://twitter.com/PeckShieldAlert/status/1630027178426642432)
 
 ## Security Failure Causes
 
-The main reason for the Lendhub attack was the failure to remove a deprecated token from the market properly. This vulnerability allowed attackers to exploit the protocol's smart contracts and steal funds.
+- **Old and new IBSV tokens exist simultaneously:** This made it possible to mint and redeem tokens on the old market, while simultaneously borrowing money against them on the new market
```
Comment (JediFaust) [inline content/attacks/posts/2023-01-12-Lendhub.md:36]: The bullet point should be common cause, I would suggest `Dev Team Negligence`. And mention that main reason was overlooked deprecated token from the market, in the description itself.

**Review by JediFaust — CHANGES_REQUESTED** [review]

**Review by JediFaust — APPROVED** [review]

**Comment by albina-at-inca** [comment]
@svg-arch please email your BTC address to wiki.challenge.bounties@dn.institute including QR code.

**Pull request merged**